tox.ini

```diff
@@ -1,45 +1,20 @@
 [tox]
-envlist = {py27,py36}-django111,py36-django20
-
-[doc8]
-max-line-length = 120
-
-[pycodestyle]
-exclude = .git,.tox,migrations
-max-line-length = 120
-
-[pydocstyle]
-; D101 = Missing docstring in public class
-; D200 = One-line docstring should fit on one line with quotes
-; D203 = 1 blank line required before class docstring
-; D212 = Multi-line docstring summary should start at the first line
-; D215 = Section underline is over-indented (numpy style)
-; D404 = First word of the docstring should not be This (numpy style)
-; D405 = Section name should be properly capitalized (numpy style)
-; D406 = Section name should end with a newline (numpy style)
-; D407 = Missing dashed underline after section (numpy style)
-; D408 = Section underline should be in the line following the section’s name (numpy style)
-; D409 = Section underline should match the length of its name (numpy style)
-; D410 = Missing blank line after section (numpy style)
-; D411 = Missing blank line before section (numpy style)
-; D412 = No blank lines allowed between a section header and its content (numpy style)
-; D413 = Missing blank line after last section (numpy style)
-; D414 = Section has no content (numpy style)
-ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
-match-dir = (?!migrations)
-
-[pytest]
-DJANGO_SETTINGS_MODULE = test_settings
-addopts = --cov super_csv --cov-report term-missing --cov-report xml
-norecursedirs = .* docs requirements
+envlist = py{27,35}-django111-drf{36,37,38,39,latest},py35-django{20,21,22}-drf{36,37,38,39,latest}
```

Comment:
We don't need the drf36 entries, do we?  Not used in Travis, and we're already on 3.7.7 in edx-platform.